### PR TITLE
layout page design

### PR DESCRIPTION
**Pull request by Loges2108**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,31 @@
-import Navbar from "./common/component/Navbar"
-
+import { Routes, Route, BrowserRouter as Router } from "react-router-dom";
+import { ThemeProvider } from '@mui/material';
+import theme from './theme/theme'; 
+import Layout from "./common/component/layout/Layout";
+import Home from "./pages/home/Home";
+import Bracelates from "./pages/bracelets/Bracelates";
```
Comment (sathyaprabaS) [inline src/App.tsx:6]: remove unwanted Import

File: src/App.tsx [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+          <Route path={paths.ROOT} element={<Layout />}>
+            <Route index element={< Home/>} />
+            <Route path={paths.EARRINGS} element={<EarRings />} />
+            <Route path={paths.BRACELETS} element={<Bracelates />} />
+            <Route path={paths.NECKLACES} element={<Necklaces />} />
+            <Route path={paths.BESTSELLER} element={<BestSelller />} />
+            <Route path={paths.FAQABOUT} element={<FAQ/>} />
+          </Route>
+        </Routes>
+      </Router>
```
Comment (sathyaprabaS) [inline src/App.tsx]: why use Router

File: src/common/component/navbar/Nav.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+import NavbarDrawer from '../../../drawer/NavDrawer';
+import SearchDrawer from '../../../drawer/SearchDrawer';
+import CartDrawer from '../../../drawer/CartDrawer';
+
+const Navbar: React.FC = () => {
+  const isMobileView = useMediaQuery("(max-width:1000px)");
+
+  const [drawerOpen, setDrawerOpen] = useState(false);
+  const [searchDrawerOpen, setSearchDrawerOpen] = useState(false);
+  const [cartDrawerOpen, setCartDrawerOpen] = useState(false);
```
Comment (sathyaprabaS) [inline src/common/component/navbar/Nav.tsx]: give proper Name

File: src/common/component/navbar/Nav.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+  const handleDrawerOpen = (e: React.MouseEvent) => {
```
Comment (sathyaprabaS) [inline src/common/component/navbar/Nav.tsx]: why use Mouse Event

File: src/drawer/CartDrawer.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
+// CartDrawer.tsx
+
+import React from 'react';
+import { Drawer, Grid, IconButton, Typography, Button } from '@mui/material';
+import { Close } from '@mui/icons-material';
+import { CartDrawerProps } from '../interface/type';
+import useStyles from '../styles/cartDrawer'; 
+
+const CartDrawer: React.FC<CartDrawerProps> = ({ open, onClose }) => {
```
Comment (sathyaprabaS) [inline src/drawer/CartDrawer.tsx]: no use react.FC

File: src/common/component/navbar/Nav.tsx [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+                  <ShoppingBasketIcon />
+                </Badge>
+              </IconButton>
+              </Box>
+            </Grid>
+    </Grid>
+    </Toolbar>
+    </AppBar>
+    <Divider sx={{ width: '80%', margin: 'auto' }} />
+    <NavbarDrawer open={drawerOpen} onClose={handleDrawerClose}/>
```
Comment (sathyaprabaS) [inline src/common/component/navbar/Nav.tsx:124]: no use component use the icon

File: src/common/component/navbar/Nav.tsx [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+    <SearchDrawer open={searchDrawerOpen} onClose={handleSearchDrawerClose}/>
```
Comment (sathyaprabaS) [inline src/common/component/navbar/Nav.tsx:125]: same previous comment

File: src/App.tsx (hunk 1 above)
Comment (Loges2108) [reply src/App.tsx:6]: it must use in app.tsx.no un wanted import

File: src/App.tsx (hunk 2 above)
Comment (Loges2108) [reply src/App.tsx]: instead of router i use browser router

File: src/common/component/navbar/Nav.tsx (hunk 3 above)
Comment (Loges2108) [reply src/common/component/navbar/Nav.tsx]: i give mybagdraweropen

File: src/common/component/navbar/Nav.tsx (hunk 4 above)
Comment (Loges2108) [reply src/common/component/navbar/Nav.tsx]: i remove the mouse event

File: src/drawer/CartDrawer.tsx (hunk 5 above)
Comment (Loges2108) [reply src/drawer/CartDrawer.tsx]: i remove it

**Pull request merged**